### DISCUSSION TITLE
libvips online meet-up, Thursday 9th December 2021, 19:00 UTC

### DISCUSSION THREAD

**Discussion by lovell** [libvips/libvips#2550] [General]
Given travel restrictions and rising Covid cases, rather than having an annual-ish gathering for pizza in London, we're planning an online event for anyone interested in libvips and its ecosystem. Hopefully this means many more people than usual will be able to join.

--

To access this event:

https://us02web.zoom.us/j/87402721287

Meeting ID: 874 0272 1287

Use the 6 letter name of this repo, all lower case, as the passcode.

--

Please feel free to add comments should there be any topics you'd like us to discuss.

Start time: 11am PST / 2pm EST / 7pm GMT / 8pm CET

**Comment by jcupitt** [comment]
I'll be there, thanks for suggesting this Lovell.

**Comment by le0daniel** [comment]
Would love to join, even though i m not an active contributor 😃

**Comment by jmreid** [comment]
Some Shopify folks would love to join in!

**Reply by sbfaulkner** [reply]
what he said! :-)

**Reply by lovell** [reply]
Great, all welcome. Would someone from Shopify like to do a quick talk at this event about the recent migration to libvips and how it is being used at scale?

**Reply by jmreid** [reply]
Sure! I'm happy to do a quick talk and intro for the Shopify team.

**Comment by jcupitt** [comment]
I'm sure no one will have the stomach for many presentations, but I could offer five minutes on where libvips came from, if there's interest.

I suppose someone could do five minutes on where it's going? Or maybe that's better left to informal discussion.

**Comment by dloebl** [comment]
Thanks for organizing everything! 
My brother (@MCLoebl) and I will join, too. If it is of interest, we could show a few slides on how we want to reduce the file size of GIFs encoded by [cgif](https://github.com/dloebl/cgif) (GIF encoder used by libvips).

**Reply by jcupitt** [reply]
Sure, that's be great @dloebl, thanks!

**Comment by kmartinez** [comment]
Has someone chosen a platform? Zoom/Teams/whatever?

**Reply by lovell** [reply]
I have a Zoom "Pro" account we can use that is limited to 100 attendees, which might be enough, but perhaps something with a greater limit and more features etc. would be better. Does Southampton Uni provide anything to staff?

**Reply by kmartinez** [reply]
sounds perfect! we have Teams - I think limited to 250 - but I think zoom is pretty nice

**Reply by jcupitt** [reply]
Hehe I think 10 is likely to be enough, but I suppose you never know.

**Reply by kmartinez** [reply]
as I have a BT engineer coming that afternoon - probably safer to go with Zoom in case I'm left stranded

**Comment by lovell** [comment]
To access this event:

https://us02web.zoom.us/j/87402721287

Meeting ID: 874 0272 1287

Use the 7 letter name of this repo, all lower case, as the passcode.

**Reply by lovell** [reply]
Thanks everyone for attending and contributing today, great to meet you all!

**Reply by jcupitt** [reply]
And thank you for organising this! I thought it went well.

**Reply by le0daniel** [reply]
Thx for organizing and your great work on libvips!
